package.json

```diff
@@ -21,7 +21,7 @@
     "angular-material": "1.0.7",
     "angular-messages": "1.5.3",
     "angular-ui-router": "0.2.18",
-    "express": "4.14.0",
+    "express": "4.15.2",
```

Comment:
ExpressJS is not used by the fh-wfm-workorder-angular module. This can be removed as a dependency.